.cloudtest.yaml

```diff
@@ -24,6 +24,11 @@ executions:
     root: ./test/integration
     timeout: 300
     cluster-count: 1
+    cluster-selector:
+      - gke
+    only-run:
+      - TestForwarderHealLocal
+      - TestNSMDDP
```

Comment:
Does this pass with the full suite of tests?

Reply:
Haven't checked yet, the PR is just a sanity check for now